docs/Modeling/EspecificacaoSuplementar.md

```diff
@@ -0,0 +1,65 @@
+# Especificação Suplementar
+
+Esta especificação tem por finaliadade apresentar os [requisitos não funcionais](/Modeling/objeto?id=requisito-não-funcional) do sistema de controle de [estoque](/Modeling/objeto?id=estoque), Stock. Será apresentada uma ambientação do projeto e, em seguida, a exposição de [requisitos suplementares](/Modeling/objeto?id=requisito-não-funcional) distribuídos em usabilidade, confiabilidade, desempenho, compatibilidade, suportabilidade, interfaces, requisitos de licenciamento e observações de copyright e outras.
+
+## Histórico de Revisões
+|     Data   | Versão |                  Descrição                | Autor(es)  |
+|:----------:|:------:|:-----------------------------------------:|:----------:|
+| 21/09/2020 |   1.0  | Criação do documento e adição da técnica. | Pedro Igor |
+| 21/09/2020 |   1.1  | Linkagem dos léxicos.                     | Pedro Igor |
+
+## Funcionalidade
+A aplicação [web](/Modeling/objeto?id=web-app), Stock deverá ser capaz de gerenciar o [estoque](/Modeling/objeto?id=estoque) de [micro e pequenas empresas](/Modeling/objeto?id=micro-empresa). Um [sócio ou dono](/Modeling/objeto?id=owner) de um negócio terá [privilégios](Modeling/objeto?id=papéis-dos-colaboradores) para criação e adição de funcionários dentro da aplicação que terão os [papéis]((Modeling/objeto?id=papéis-dos-colaboradores)) de [cadastrar](/Modeling/verbo?id=cadastrar-produto) [produtos](/Modeling/objeto?id=produto), alimentar [estoque](/Modeling/objeto?id=estoque) e dar [baixa](/Modeling/verbo?id=baixa-em-produto) no [estoque](/Modeling/objeto?id=estoque) ao realizar uma venda. A aplicação permitirá realizar essas ações de forma remota e com atualização instantânea. Além disso, será possível fazer buscas por [produtos](/Modeling/objeto?id=produto), analisar o [histórico](/Modeling/objeto?id=log) de [fluxo](/Modeling/verbo?id=fluxo) de [estoque](/Modeling/objeto?id=estoque), [consultar gráficos](/Modeling/verbo?id=consultar-gráfico) a respeito desse [fluxo](/Modeling/verbo?id=fluxo) e [exportar](/Modeling/verbo?id=exportação) e [importar](/Modeling/verbo?id=importação) documentos de [planilhas](/Modeling/objeto?id=planilha).
```

Comment:
O link de papéis está quebrado e acho que nem se encaixa nesse léxico, porque papéis dos colaboradores é a função que desempenham na empresa